src/alembic_utils/replaceable_entity.py

```diff
@@ -173,15 +174,45 @@ def get_required_migration_op(
         return CreateOp(self)
 
 
+class EntityWithOp(NamedTuple):
+    entity: ReplaceableEntity
+    op: ReversibleOp
+
+
+def depends_on_expanded(entity: ReplaceableEntity) -> set[ReplaceableEntity]:
```

Comment:
I think we need some unit tests to cover this function.

Reply:
added